app/views/setup/welcome.html.slim

```diff
@@ -8,6 +8,9 @@
 h1 Initial CaaSP Configuration
 = form_for :settings, url: setup_path, method: :put do |f|
   .form-group
+    = f.label :dashboard, "Dashboard location"
+    = f.text_field :dashboard, value: (Pillar.value(pillar: :dashboard) || request.host), class: "form-control"
```

Comment:
Its implication highly depends on the contents of `request.host`, while `localhost` will always fail, any other resolvable name or reachable ip address by the cluster will work fine. What I mean: using as dashboard `request.host` isn't bad per-se, it highly depends on the `request.host` actually (e.g. `localhost` will always be a bad idea, for example).